src/toil/test/cwl/cwlTest.py

```diff
@@ -414,46 +401,27 @@ def tearDown(self):
             shutil.rmtree(self.outDir)
         unittest.TestCase.tearDown(self)
 
+    def refresh_deleted_fixtures(self):
+        # TODO: Prevent Toil from deleting these?
```

Comment:
Great catch, thanks! Can we get an issue to track why this is needed?

Reply:
Definitely!  I've created: https://github.com/DataBiosphere/toil/issues/3318